src/utilities/idf/WorkspaceObject.cpp

```diff
@@ -188,7 +188,7 @@ namespace detail {
     return IdfObject_Impl::getString(index, returnDefault, returnUninitializedEmpty);
   }
 
-  boost::optional<std::string> WorkspaceObject_Impl::getField(unsigned index) const {
+  boost::optional<std::string> WorkspaceObject_Impl::getField(unsigned index, bool returnDefault) const {
```

Comment:
for WorkspaceObject, it does **IdfObject_Impl::getString** for any non-source field, and grabs the handle otherwise